docs/wiki/Solution creation.md

```diff
@@ -43,12 +44,37 @@ When it comes to deploying multi-module solutions (applications/workloads/enviro
 
    <img src="./media/SolutionCreation/templateOrchestration.png" alt="Template orchestration" height="250">
 
-- **_Pipeline-orchestration_**: This approach uses the platform specific pipeline capabilities (for example, pipeline jobs) to trigger the deployment of individual modules, where each job deploys one module. By defining dependencies in between jobs you can make sure your resources are deployed in order. Parallelization is achieved by using a pool of pipeline agents that run the jobs, while accounting for all dependencies defined.
+    _Advantages_
```

Comment:
Should we add that the deployment is faster?

Reply:
added